Resources/Shaders/InvertMaterial.gdshader

```diff
@@ -0,0 +1,14 @@
+shader_type spatial;
+render_mode unshaded;
+
+#include "res://Resources/ShaderIncs/ResoniteBlendMode.gdshaderinc"
```

Comment:
If a given Unity shader contains the ``_Cull`` parameter that is passed into ``Cull[_Cull]`` (which Invert does), it should include ``ResoniteCullMode``